Files/UserControls/ModernNavigationToolbar.xaml

```diff
@@ -523,6 +524,21 @@
                             </StackPanel>
                         </Grid>
                     </DataTemplate>
+                    <ResourceDictionary.ThemeDictionaries>
+                        <ResourceDictionary x:Key="Light">
+                            <RevealBorderBrush x:Key="TransparentGridRevealBorderBrush"
+                                               TargetTheme="Light"
+                                               Color="Transparent"
+                                               FallbackColor="Transparent" />
+                        </ResourceDictionary>
+                        <ResourceDictionary x:Key="Dark">
+                            <RevealBorderBrush x:Key="TransparentGridRevealBorderBrush"
+                                               TargetTheme="Dark"
+                                               Color="Transparent"
+                                               FallbackColor="Transparent" />
+                        </ResourceDictionary>
+                        <ResourceDictionary x:Key="HighContrast" />
+                    </ResourceDictionary.ThemeDictionaries>
```

Comment:
Instead of reveal, the referenced issue proposed changing the background of the textbox when hovering over it.

Reply:
Made changes as you requested.